services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet[CinemaHall]:
+    result = CinemaHall.objects.all()
+    return result
```

Comment:
There is no need to allocate `result` variable, return queryset directly. Change approach in other places as well (e.g. `get_movie_by_id` method)